docs/source/installation.rst

```diff
@@ -136,52 +136,47 @@ Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
 
     .. tabs::
 
-        .. tab:: CPU Support
-
-            If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:
+        .. tab:: PyPI Installation
```

Comment:
What do you think about linking these two sets of tabs? 

Sphinx-tabs has a [Group Tabs](https://sphinx-tabs.readthedocs.io/en/latest/#group-tabs) feature that lets you change all the linked tabs at once. (We could also do that for the Mac/Linux and Windows tabs, too.)

Reply:
SICK yo. That is such a cool trick! OMG. So cool. I will definitely use it and adopt it for others. TIL!